src/include/daos/drpc_modules.h

```diff
@@ -53,12 +54,8 @@ enum drpc_mgmt_method {
 	DRPC_METHOD_MGMT_POOL_QUERY             = 223,
 	DRPC_METHOD_MGMT_POOL_SET_PROP          = 224,
 	DRPC_METHOD_MGMT_PING_RANK              = 225,
-	DRPC_METHOD_MGMT_REINTEGRATE            = 226,
 	DRPC_METHOD_MGMT_CONT_SET_OWNER         = 227,
-	DRPC_METHOD_MGMT_EXCLUDE                = 228,
-	DRPC_METHOD_MGMT_EXTEND                 = 229,
 	DRPC_METHOD_MGMT_POOL_EVICT             = 230,
-	DRPC_METHOD_MGMT_DRAIN                  = 231,
```

Comment:
What's the motivation for renumbering these? It shouldn't cause any problems since daos_server and daos_engine must be the same version, but... why? If you're just standardizing the naming without changing the meaning, IMO it's better to rename in place, with the same numbers.

Reply:
done